packages/block-library/src/block/edit.js

```diff
@@ -109,7 +109,13 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 					<ToolbarGroup>
 						<ToolbarButton
 							onClick={ () => convertBlockToStatic( clientId ) }
-							label={ __( 'Convert to regular blocks' ) }
+							label={ sprintf(
+								/* translators: %s: properly pluralized form of block */
+								__( 'Convert to regular %s' ),
+								blocks[ 0 ]?.innerBlocks.length > 1
+									? 'blocks'
+									: 'block'
```

Comment:
The `block(s)` part of the string also needs to be translated. I would change the logic to something like this:

```js
count  > 1 ? __( 'Convert to regular blocks' ) : __( 'Convert to regular block' )
```

Reply:
The [@wordpress/i18n](https://www.npmjs.com/package/@wordpress/i18n) package supports the [_n() function](https://developer.wordpress.org/reference/functions/_n/) which _translates and retrieves the singular or plural form based on the supplied number_. Using that function, the code would be:
`_n( 'Convert to regular block', 'Convert to regular blocks', count );`

Reply:
We shouldn't use `_n` in cases of "one item or more than items". See https://developer.wordpress.org/reference/functions/_n/#comment-2397.